### PR TITLE
added upgrade 2.0.0 to 2.0.1

### PR DESCRIPTION
**Pull request by cooper-lzy**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -213,7 +213,8 @@ nav:
           - 源码编译安装: 4.deployment-and-installation/2.compile-and-install-nebula-graph/1.install-nebula-graph-by-compiling-the-source-code.md
           - RPM/DEB包安装: 4.deployment-and-installation/2.compile-and-install-nebula-graph/2.install-nebula-graph-by-rpm-or-deb.md
       - 部署Nebula Graph集群: 4.deployment-and-installation/deploy-nebula-graph-cluster.md
-      - 升级Nebula Graph: 4.deployment-and-installation/3.upgrade-nebula-graph.md
+      - 升级历史版本Nebula Graph: 4.deployment-and-installation/3.upgrade-nebula-graph.md
```
Comment (whitewum) [inline mkdocs.yml:216]: 升级历史版本到Nebula Graph v2.0.0

File: docs-2.0/4.deployment-and-installation/5.upgrade-nebula-from-200-to-201.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Nebula Graph v2.0.0升级至v2.0.1
+
+Nebula Graph v2.0.0升级至v2.0.1，只需要使用v2.0.1的RPM/DEB包进行升级操作即可，或者[编译v2.0.1](2.compile-and-install-nebula-graph/1.install-nebula-graph-by-compiling-the-source-code.md)之后重新安装。
+
+!!! note
+    如果Nebula Graph版本过低，需要先升级至v2.0.0，详情请参见[升级 Nebula Graph 历史版本至 v2.0.0](3.upgrade-nebula-graph.md)。
```
Comment (whitewum) [inline docs-2.0/4.deployment-and-installation/5.upgrade-nebula-from-200-to-201.md:6]: one linebreak?

**Review by whitewum — APPROVED** [review]
minor and ok

File: docs-2.0/4.deployment-and-installation/5.upgrade-nebula-from-200-to-201.md (hunk 2 above)
Comment (randomJoe211) [reply docs-2.0/4.deployment-and-installation/5.upgrade-nebula-from-200-to-201.md:6]: Not necessary.

**Review by randomJoe211 — APPROVED** [review]

**Review by FengUyu — APPROVED** [review]

**Pull request merged**